submissions/description/cross_site_scripting_xss/off_domain/data_uri/template.md

```diff
@@ -0,0 +1,30 @@
+# Cross-Site Scripting Off Domain via Data URI
+
+## Overview of the Vulnerability
+
+Cross-Site Scripting (XSS) is a type of injection attack where malicious JavaScript is injected into a website. When a user visits the affected web page, the Javascript executes within that user’s browser in the context of the domain. XSS can be found in this application which allows an attacker to input data into the URL that can be interpreted as a JavaScript payload. The data is then executed in the context of a domain which is off the primary domain. This carries the risk of an attacker being able to trigger an exploit on a seperate domain. By controlling code that is executed within a user’s browser, an attacker could carry out any actions that the user is able to perform, including accessing any of the user's data and modifying information within the user’s permissions.
```

Comment:
This feels very wordy, splitting this into 2 paragraphs will improve readability a whole lot

Reply:
Good pick up, thanks